src/VisualStudio/Core/Impl/CodeModel/FileCodeModel_Events.cs

```diff
@@ -136,7 +136,7 @@ private EnvDTE80.vsCMChangeKind ConvertToChangeKind(CodeModelEventType eventType
             return result;
         }
 
-        private void GetElementsForCodeModelEvent(CodeModelEvent codeModelEvent, out EnvDTE.CodeElement element, out object parentElement)
+        internal void GetElementsForCodeModelEvent(CodeModelEvent codeModelEvent, out EnvDTE.CodeElement element, out object parentElement)
```

Comment:
Were there any other tests that verified this method via some other entry point? If this really needs to be made internal for testing, can you mark it as ```// internal for testing```?